core/trino-main/src/main/java/io/trino/sql/planner/PlanOptimizers.java

```diff
@@ -378,6 +378,25 @@ public PlanOptimizers(
                 new PushDownDereferencesThroughRowNumber(typeAnalyzer),
```

Comment:
> Allows pushdown of multiple distinct aggregations as long as
only one non-count aggregation is distinct.

What is the difference between one distinct non-count aggregation and multiple distinct non-count aggregations?

Reply:
Maybe the phrasing is bad, but I just meant that in that case aggregations distinct are treated the same way they are today, which falls to `OptimizeMixedDistinctAggregations`